src/app/components/drawer/Projects/ProjectsComponent.js

```diff
@@ -215,70 +215,136 @@ const ProjectsComponent = ({
 
   return (
     <React.Fragment>
+      <div className={styles.listTitle}>
+        Tipline
+      </div>
       <List dense disablePadding className={[styles.listWrapper, 'projects-list'].join(' ')}>
         { saving && <div className={styles.listMask} /> }
         {/* All items */}
         <ListItem
           button
           onClick={handleAllItems}
-          className={['projects-list__all-items', styles.listItem, (activeItem.type === 'all-items' ? styles.listItem_active : '')].join(' ')}
+          className={['projects-list__all-items', styles.listItem, styles.listItem_containsCount, (activeItem.type === 'all-items' ? styles.listItem_active : '')].join(' ')}
         >
-          <div className={styles.listIcon}>
-            <CategoryIcon />
-          </div>
+          <CategoryIcon className={styles.listIcon} />
           <ListItemText disableTypography className={styles.listLabel}>
             <FormattedMessage tagName="span" id="projectsComponent.allItems" defaultMessage="All" description="Label for the 'All items' list displayed on the left sidebar" />
           </ListItemText>
-          <ListItemSecondaryAction disableTypography title={team.medias_count} className={styles.listItemCount}>
-            {team.medias_count}
+          <ListItemSecondaryAction disableTypography className={styles.listItemCount}>
+            <small>
+              {team.medias_count}
+            </small>
           </ListItemSecondaryAction>
         </ListItem>
 
         { team.smooch_bot &&
           <ListItem
             button
             onClick={() => { handleSpecialLists('tipline-inbox'); }}
-            className={['projects-list__tipline-inbox', styles.listItem, (activeItem.type === 'tipline-inbox' ? styles.listItem_active : '')].join(' ')}
+            className={['projects-list__tipline-inbox', styles.listItem, styles.listItem_containsCount, (activeItem.type === 'tipline-inbox' ? styles.listItem_active : '')].join(' ')}
           >
-            <div className={styles.listIcon}>
-              <InboxIcon />
-            </div>
+            <InboxIcon className={styles.listIcon} />
             <ListItemText disableTypography className={styles.listLabel}>
               <FormattedMessage tagName="span" id="projectsComponent.tiplineInbox" defaultMessage="Inbox" description="Label for a list displayed on the left sidebar." />
             </ListItemText>
+            <ListItemSecondaryAction disableTypography className={styles.listItemCount} />
```

Comment:
for reviews info:
we will be adding counts to all items in the future, but you will see some empty count items like this because for some reason, if you don't include it, then the list item will render very differently from material and it makes styling more challenging. 🤷